openedx/features/caliper_tracking/transformers/enrollment_transformers.py

```diff
@@ -24,7 +24,8 @@ def edx_course_enrollment_activated(current_event, caliper_event):
             'extensions': {
                 'mode': current_event['event']['mode'],
                 'course_id': current_event['context']['course_id'],
-                'user_id': caliper_event['extensions']['extra_fields'].pop('user_id')
+                'user_id': caliper_event['extensions']['extra_fields'].pop(
+                    'user_id')
```

Comment:
What is this ?

Reply:
`user_id` was placed in `caliper_event` in `base_transformer` but i moved `user_id` to `extensions of object`, this is why i had to place and pop `user_id`, which i preferred to be done simultaneously.

Reply:
I mean why have you indented `'user_id')` like that?

Reply:
It's due to pep8 issue.